Dreary/README.md

```diff
@@ -17,15 +17,23 @@
 ### Golden
 ![Golden Screenshot](golden.png)
 
+### Graytone-Blue
+![Graytone-Blue Screenshot](graytone-blue.png)
+
+### Graytone-Red
+![Graytone-Red Screenshot](graytone-red.png)
+
+
```

Comment:
There is no preview for Kimbie-Dark

Reply:
@morpheusthewhite Removed theme completely, didn't like it